library/psa_crypto.c

```diff
@@ -545,6 +545,68 @@ static inline size_t psa_get_key_slot_bits( const psa_key_slot_t *slot )
     return( slot->attr.bits );
 }
 
+/** Return the output MAC length of a MAC algorithm, in bytes
+ *
+ * \param[in] algorithm     The specific MAC algorithm
+ * \param[in] key_type      The key type of the key to be used with the
+ *                          \p algorithm.
+ * \param[out] length       The calculated output length of the given MAC
+ *                          \p algorithm when used with a key corresponding to
+ *                          the given \p key_type
+ *
+ * \retval #PSA_SUCCESS
+ *         The \p length has been successfully calculated
+ * \retval #PSA_ERROR_NOT_SUPPORTED
+ *         \p algorithm is a MAC algorithm, but ubsupported by this PSA core.
+ * \retval #PSA_ERROR_INVALID_ARGUMENT
+ *         \p algorithm is not a valid, specific MAC algorithm or \p key_type
+ *         describes a key incompatible with the specified \p algorithm.
+ */
+static psa_status_t psa_get_mac_output_length( psa_algorithm_t algorithm,
```

Comment:
`PSA_MAC_LENGTH` takes in key_bits as an argument, even though it's fully unused in its current definition. If I re-use that macro here, are you then on board with calling it with a bogus (i.e. 0) value for key_bits? I don't think `psa_get_mac_output_length` needs to require key_bits in addition to key_type.

Reply:
Ok with a comment. The public-facing macro takes `bits` in case we later wanted to add an algorithm for which the MAC length is tied to the key length. For an internal function, it doesn't matter.

Reply:
Done